packages/inscription-view/src/components/browser/type/TypeBrowser.tsx

```diff
@@ -92,6 +91,7 @@ const TypeBrowser = ({ value, onChange, onDoubleClick, initSearchFilter, locatio
           : type.fullQualifiedName.includes('ivy')
           ? IvyIcons.Ivy
           : IvyIcons.DataClass,
+        description: dataClasses.find(dc => dc.fullQualifiedName === type.fullQualifiedName)?.description || '',
```

Comment:
on line 89 we already lookup this dataclass to define the icon ... can we hold this variable and re-use it ? :thinking:

Reply:
```tsx
const mappedAllTypes: TypeBrowserObject[] = allDatatypes.map<TypeBrowserObject>(type => {
  const dataClass = dataClasses.find(dc => dc.fullQualifiedName === type.fullQualifiedName);
  return {
        icon: dataClass ? IvyIcons.LetterD : type.fullQualifiedName.includes('ivy') ? IvyIcons.Ivy : IvyIcons.DataClass,
        description: dataClass?.description || '',

```

Reply:
thanks: this one missing brace was keeping me back :sweat_smile: